packages/utils/src/lib/utils.spec.ts

```diff
@@ -136,69 +47,3 @@ describe('distinct', () => {
     ]);
   });
 });
-
-describe('toArray', () => {
-  it('should transform non-array value into array with single value', () => {
-    expect(toArray('src/**/*.ts')).toEqual(['src/**/*.ts']);
-  });
-
-  it('should leave array value unchanged', () => {
-    expect(toArray(['*.ts', '*.js'])).toEqual(['*.ts', '*.js']);
-  });
-});
```

Comment:
The `toArray` function is still in `utils.ts`. Please double-check you aren't deleting any tests :pray: 

(Overall I like the splitting of into multiple files, though :+1: )

Reply:
It is still here. I sorted the tests in the same order as in the file. Will mention it next time i th ePR comment.

Reply:
Oh, I see now you just changed the order. Sorry, the diff confused me, false alarm :man_facepalming: